Extension/tsconfig.json

```diff
@@ -2,25 +2,29 @@
     "compilerOptions": {
         "module": "commonjs",
         "target": "ES2022",
-        "outDir": "out",
+        "outDir": "dist",
         "lib": [
             "ES2022",
             "dom"
         ],
-        "sourceMap": true,
+        "inlineSourceMap": true,
         "rootDir": ".",
         "removeComments": true,
         "noUnusedLocals": true,
         "strictNullChecks": true,
         "strict": true,
-        "forceConsistentCasingInFileNames": true,
+        "forceConsistentCasingInFileNames": true
     },
     "include": [
-        "src/**/*.ts",
+        "test/**/*.ts",
+        "ui/**/*.ts",
         "./vscode*.d.ts",
-        "./node_modules/vscode/lib/*"
+        "src/types/*.d.ts",
+        "src/**/*.ts"
     ],
     "exclude": [
-        "node_modules/@types/eslint-scope/node_modules/@types/eslint/index.d.ts"
+        "node_modules/@types/eslint-scope/node_modules/@types/eslint/index.d.ts",
+        "test/attic/**",
```

Comment:
what is `test/attic`?  I don't see a folder with that name.

Reply:
Oh, a temp folder I put in place while I was reorganizing things. removed.